options/locale/locale_en-US.ini

```diff
@@ -142,6 +142,19 @@ confirm_delete_selected = Confirm to delete all selected items?
 name = Name
 value = Value
 
+[search]
+search = Search
+search_kind = Search %s...
+type_tooltip = Search type
+fuzzy = Fuzzy
+fuzzy_tooltip = Include results that also match the search term closely
+match = Match
+match_tooltip = Include only results that match the exact search term
+no_results = No %s matching your search term found.
```

Comment:
Not a good idea to deduplicate this term.
Or are you using it as `ctx.Locale.Tr("…no_results", ctx.Locale.Tr("…repo"))` for example?

Reply:
Yep, you did exactly that…

Reply:
This is very tricky, while it works in English it will not work in other languages (word casing, word forms etc, for example in Latvian each noun has 7 forms for each singular and multiple cases with different endings depending on the sentence where it's used), so explore labels definitely cannot be used here

Reply:
@lafriks Maybe simply do "Search..." everywhere where it's obvious?

Reply:
I would not mind from my perspective but see @fnetX comment and I agree that it's better to have specific placeholder for accessibility